app/workers/commit_monitor_handlers/commit_range/github_pr_commenter/commit_metadata_checker.rb

```diff
@@ -60,5 +61,13 @@ def check_for_usernames_in(commit, message)
         @offenses << OffenseMessage::Entry.new(:low, message, group)
       end
     end
+
+    def check_for_merge_commit(commit, merge_commit)
+      return unless merge_commit
+
+      group   = ::Branch.github_commit_uri(fq_repo_name, commit)
+      message = "Merge commit #{commit} detected.  Consider rebasing."
+      @offenses << OffenseMessage::Entry.new(:low, message, group)
```

Comment:
Does `:low` translate into ⚠️  or ❕ ?  I think I'd prefer the ⚠️  (probably same goes for `usernames` check)

Reply:
I kept the same as the username check, https://github.com/ManageIQ/miq_bot/blob/master/app/workers/commit_monitor_handlers/commit_range/github_pr_commenter/commit_metadata_checker.rb#L60

Reply:
Looks like `:low` -> :grey_exclamation: